src/services/ai-server/ai-persona-service/ai.persona.service.service.ts

```diff
@@ -67,15 +68,19 @@ export class AiPersonaServiceService {
       LogContext.PLATFORM
     );
 
-    if (aiPersonaServiceData.bodyOfKnowledgeID) {
-      this.eventBus.publish(
-        new IngestSpace(
-          savedAiPersonaService.bodyOfKnowledgeID,
-          SpaceIngestionPurpose.KNOWLEDGE,
-          savedAiPersonaService.id
-        )
-      );
-    }
+    //TODO enable again - having a bit of a race condition and the
```

Comment:
is this tracked?

Reply:
@coderabbitai can you write up a story about this for me, please?